packages/smarthr-ui/src/libs/ua.ts

```diff
@@ -1,20 +1,28 @@
 const ua = typeof window !== 'undefined' ? window.navigator.userAgent.toLowerCase() : 'SSR'
 
+const isWindows = ua.indexOf('windows') !== -1
+const isAndroid = ua.indexOf('android') !== -1
+const isMobile = ua.indexOf('mobile') !== -1
+const isFirefox = ua.indexOf('firefox') !== -1
+const isIphone = ua.indexOf('iphone') !== -1
+const isIpod = ua.indexOf('ipod') !== -1
+const isIpad = ua.indexOf('ipad') !== -1
+
 export const isIe = ua.indexOf('msie') !== -1 || ua.indexOf('trident') !== -1
```

Comment:
nits) これも正規表現にしちゃっても良かったりしますかね...？
```suggestion
export const isIe = /msie|trident/.test(ua)
```

Reply:
こちらもやる場合、別PRで... 🙏